tests/e2e/taqueria-plugin-ligo.spec.ts

```diff
@@ -0,0 +1,72 @@
+import {checkFolderExistsWithTimeout, generateTestProject} from "./utils/utils";
+import fs from "fs";
+import {execSync} from "child_process";
+
+const taqueriaProjectPath = 'e2e/auto-test-ligo-plugin';
+
+describe("E2E Testing for taqueria ligo plugin",  () => {
+
+    beforeAll(async () => {
+        await generateTestProject(taqueriaProjectPath, ["ligo"]);
+    })
+
+    test('Verify that taqueria ligo plugin can compile one contract using compile all command', async () => {
+        try {
+            // 1. Copy contract from data folder to taqueria project folder
+            execSync(`cp e2e/data/hello-tacos.mligo ${taqueriaProjectPath}/contracts`);
+
+            // 2. Run taq compile ${contractName}
+            execSync(`cd ./${taqueriaProjectPath} && taq compile`);
+
+            // 3. Verify that compiled michelson version has been generated
+            await checkFolderExistsWithTimeout(`./${taqueriaProjectPath}/artifacts/hello-tacos.tz`, 25000);
+
+        } catch(error) {
+            throw new Error (`error: ${error}`);
+        }
+
+    });
+
+    test.skip('Verify that taqueria ligo plugin can compile one contract using compile [sourceFile] command', async () => {
```

Comment:
@sinapsist does this need to be skipped currently?

Reply:
It was a bug, not it is fixed.